habitat_baselines/rl/ver/report_worker.py

```diff
@@ -0,0 +1,423 @@
+                    self.window_episode_stats[k].add_many(vs)
+
+        if rank0_only():
+            assert all_learner_metrics is not None
+            learner_metrics = cast(
+                Dict[str, float],
+                (
+                    NDArrayDict.from_tree(
+                        transpose_list_of_dicts(*all_learner_metrics)
+                    )
+                    .map(lambda n: np.mean(n))
+                    .to_tree()
+                ),
+            )
+            n_steps = int(self.num_steps_done)
+            if "reward" in self.window_episode_stats:
+                writer.add_scalar(
+                    "reward",
+                    self.window_episode_stats["reward"].mean,
+                    n_steps,
+                )
+
+            for k in self.window_episode_stats.keys():
+                if k == "reward":
+                    continue
+
+                writer.add_scalar(
+                    f"metrics/{k}", self.window_episode_stats[k].mean, n_steps
+                )
+
+            for k, v in learner_metrics.items():
+                writer.add_scalar(f"learner/{k}", v, n_steps)
+
+            writer.add_scalar(
+                "perf/fps",
+                n_steps / float(self.time_taken),
+                n_steps,
+            )
+            writer.add_scalar(
+                "perf/fps_window",
+                float(self.running_frames_window)
+                / float(self.running_time_window),
+                n_steps,
+            )
+
+        if rank0_only():
+            assert all_preemption_decider_reports is not None
+            preemption_decider_report = cast(
+                Dict[str, float],
+                NDArrayDict.from_tree(
+                    transpose_list_of_dicts(*all_preemption_decider_reports)
+                )
+                .map(lambda n: np.mean(n))
+                .to_tree(),
+            )
+            preemption_decider_report = {
+                k: v / (self.world_size if "time" in k else 1)
+                for k, v in preemption_decider_report.items()
+            }
+            for k, v in preemption_decider_report.items():
+                writer.add_scalar(f"preemption_decider/{k}", v, n_steps)
+
+        if self.n_update_reports % self.config.LOG_INTERVAL == 0:
+            if rank0_only():
+                logger.info(
+                    "update: {}\tfps: {:.1f}\twindow fps: {:.1f}\tframes: {:d}".format(
+                        self.n_update_reports,
+                        n_steps / float(self.time_taken),
+                        float(self.running_frames_window)
+                        / float(self.running_time_window),
+                        n_steps,
+                    )
+                )
+                if len(self.window_episode_stats) > 0:
+                    logger.info(
+                        "Average window size: {}  {}".format(
+                            list(self.window_episode_stats.values())[0].count,
```

Comment:
```suggestion
                            next(iter(self.window_episode_stats.values())).count,
```
This method return the first element in constant time.